src/functionalTest/groovy/com/github/spotbugs/snom/CacheabilityFunctionalTest.groovy

```diff
@@ -164,7 +168,7 @@ class CacheabilityFunctionalTest extends BaseFunctionalTest {
         return result.output.find('Build cache key for task \':spotbugsMain\' is .*')
     }
 
-    private static void initializeBuildFile(File buildDir) {
```

Comment:
This `Instant` makes Java files unique for each test case. It ensures that the first build in test cases has no build cache.